app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,25 @@
+class CommentsController < ApplicationController
+  def create
+    @user = current_user
+    @post = Post.find_by(id: params['post_id'])
+    @comment = @post.comments.new(text: comment_params[:text], user: @user)
+
+    respond_to do |format|
+      format.html do
+        if @comment.save
+          # flash[:success] = "Post created successfully"
```

Comment:
- [ ] Please do not leave commented pieces of code in your codebase, they're messy and unprofessional. You can use git history to review your previous code.

Reply:
Well solved